demo/dashboard/client/src/App.js

```diff
@@ -1,5 +1,7 @@
-import React from 'react';
-import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
+import React, { Fragment } from 'react';
+import { Route, NavLink } from 'react-router-dom';
```

Comment:
I still get the same error as before:
![router-still-failling](https://user-images.githubusercontent.com/1923476/33607836-b4808858-d9c2-11e7-9722-6b4c49c52ea3.gif)

It's basically the same approach as before with new dependencies it seems.

My suggestion would be to just use [HashRouter](https://reacttraining.com/react-router/web/api/HashRouter) ([docs](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/HashRouter.md))

It seems to work just fine with the existing `NavLink`:
![router-not-failling](https://user-images.githubusercontent.com/1923476/33607962-135f0fca-d9c3-11e7-87f8-b3a6f3bb1ae9.gif)

Of course, there will be a hash because there is no server to handle things on a back-end.

Here's the diff of the suggestion
[router.diff.txt](https://github.com/ec-europa/eubfr-data-lake/files/1531153/router.diff.txt)

You'll have to either run a local server on the `dist` version or deploy the app in "prod" s3 bucket to reproduce the issue.